README.md

```diff
@@ -0,0 +1,54 @@
+# Admission trial project
+
+> Set up a Project2-trial folder.
+
+![image](https://user-images.githubusercontent.com/66781780/124811245-4eba5700-df73-11eb-8a75-484d4cce4dcc.png)
+
+In this project, we will set up a "A copy-admission-trial-project" repository. No complex coding is required for this exercise. Our goal here is to master all of the tools and best practices we learned about in previous steps.
+
+## Built With
```

Comment:
You add comments to the changes. You basically say what you think could be improved.